pkg/crc/preflight/preflight_checks_linux.go

```diff
@@ -217,51 +216,61 @@ func fixLibvirtServiceRunning() (bool, error) {
 
 func checkMachineDriverLibvirtInstalled() (bool, error) {
 	logging.Debugf("Checking if %s is installed", libvirtDriverCommand)
+
 	// Check if crc-driver-libvirt is available
-	path, err := exec.LookPath(libvirtDriverCommand)
-	if err != nil {
-		return false, err
-	}
-	fi, err := os.Stat(path)
+	libvirtDriverPath := filepath.Join(constants.CrcBinDir, libvirtDriverCommand)
+	err := unix.Access(libvirtDriverPath, unix.X_OK)
```

Comment:
@cfergeau As per `unix` package guide it states `The primary use of this package is inside other packages that provide a more portable interface to the system, such as "os", "time" and "net". Use those packages rather than this one if you can.`. I am OK right now to using it here since this code is not going to run other than linux systems but we have to be careful which all places we should use.

Reply:
I know this is not portable, but I could not find something similar in `os` (the `stat` checks we have/had in the codebase to try and do that had various issues, in particular with 0700 files owned by $anotheruser, or owned by the current user)

Reply:
@cfergeau I didn't get the ` in particular with 0700 files owned by $anotheruser, or owned by the current user` this one you mean os.Stat doesn't provide correct info around it?

Reply:
I was somehow referring to the code I'm removing, it's testing if the executable bit is set for either group or other. This test will unexpectedly fail if the file is 700 and owned by the current user, or and it will succeed when it should not if the file is 070 but the user is not part of the appropriate group. I only wanted to point out it's not simple to achieve the same as unix.Access using os.Stat.